chromium/background.js

```diff
@@ -618,3 +627,38 @@ chrome.runtime.onConnect.addListener(function (port) {
     });
   }
 });
+
+// This is necessary for communication with the popup in Firefox Private
+// Browsing Mode, see https://bugzilla.mozilla.org/show_bug.cgi?id=1329304
+chrome.runtime.onMessage.addListener(function(message, sender, sendResponse){
```

Comment:
That's right.  There is a separate IPC API for inter-extension, `chrome.runtime.onMessageExternal`.  See https://developer.chrome.com/extensions/messaging#external